src/fetchtastic/downloader.py

```
@@ -1192,44 +1302,57 @@ def extract_version(dir_name: str) -> str:
+    with ThreadPoolExecutor(max_workers=5) as executor:
+        timestamps = list(
+            executor.map(
+                lambda h: (
+                    get_commit_timestamp("meshtastic", "firmware", h) if h else None
+                ),
+                commit_hashes,
+            )
         )
 
-    target_prereleases = repo_prerelease_dirs[:1]
+    dirs_with_timestamps = list(zip(matching_prerelease_dirs, timestamps))
```

Ruff (0.14.1)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)